tests/spark/test_spark_model_export.py

```diff
@@ -92,6 +92,49 @@ def spark_model_iris(spark_context):
                               predictions=preds)
 
 
+@pytest.fixture(scope="session")
+def spark_model_transformer(spark_context):
+    iris = datasets.load_iris()
```

Comment:
Nit: We can probably DRY this out a bit. Consider defining a fixture that creates pandas and spark representations of the iris data (the fixture can return a tuple).